src/aspire/basis/fpswf_2d.py

```diff
@@ -111,40 +112,35 @@ def evaluate_t(self, images):
         """
         Evaluate coefficient vectors in PSWF basis using the fast method
 
-        :param images: coefficient array in the standard 2D coordinate basis
-            to be evaluated.
-        :return : The evaluation of the coefficient array in the PSWF basis.
+        :param images: Image stack in the standard 2D coordinate basis
+        :return : coefficient array in the PSWF basis
         """
 
-        images = np.moveaxis(images, 0, -1)  # RCOPT
+        if not isinstance(images, Image):
+            logger.warning(
+                "FPSWFBasis2D.evaluate_t expects Image instance,"
+                " attempting conversion."
+            )
+            images = Image(images)
```

Comment:
Should we pass this an array in testing?

Reply:
yes, we should, I was being lazy. I'll do it.